doc/manual/user-11-prometheus.md

```diff
@@ -258,4 +258,104 @@ Number of sockets the client used
 
 ## pgagroal_vault_self_sockets
 
-Number of sockets used by pgagroal-vault itself
\ No newline at end of file
+Number of sockets used by pgagroal-vault itself
+
+# Prometheus Metrics for OS Kernel Version
+
+pgagroal exposes kernel version information through three distinct Prometheus metrics, depending on the underlying operating system:
+
+---
+
+| Operating System           |    Metric Name       |       Included Labels                | 
+|----------------------------|----------------------|--------------------------------------|
+| **Linux**                  | `pgagroal_os_linux`  | `os`, `major`, `minor`, `patch`      |
+| **macOS**                  | `pgagroal_os_macos`  | `os`, `major`, `minor`, `patch`      |
+|**BSD (FreeBSD, OpenBSD)**      | `pgagroal_os_bsd`    | `os`, `major`, `minor`               |
+
+---
+
+
+## pgagroal_os_linux
+
+The `pgagroal_os_linux` metric reports the kernel version for **Linux-based** systems. It includes the following labels:
+
+- `os`: The name of the operating system.
+- `major`: The major version of the kernel.
+- `minor`: The minor version of the kernel.
+- `patch`: The patch version of the kernel.
+
+###  Conditions for Exposure:
+- The metric is exposed **only if the kernel version retrieval succeeds**.
+- If kernel version retrieval fails, **no metric will be emitted**.
+
+### Example Output:
+```text
+# HELP pgagroal_os_linux Kernel version as major.minor.patch
+# TYPE pgagroal_os_linux gauge
+pgagroal_os_linux{os="Linux", major="6", minor="13", patch="4"} 1
+```
+
+---
+
+## pgagroal_os_macos
+
+The `pgagroal_os_macos` metric reports the kernel version for **macOS-based** systems. It includes the same labels as Linux:
+
+- `os`: The name of the operating system.
+- `major`: The major version of the kernel.
+- `minor`: The minor version of the kernel.
+- `patch`: The patch version of the kernel.
+
+###  Conditions for Exposure:
+- The metric is exposed **only if the kernel version retrieval succeeds**.
+- If kernel version retrieval fails, **no metric will be emitted**.
+
+### Example Output:
+```text
+# HELP pgagroal_os_macos Kernel version as major.minor.patch
+# TYPE pgagroal_os_macos gauge
+pgagroal_os_macos{os="macOS", major="23", minor="3", patch="0"} 1
+```
+
+---
+
+## pgagroal_os_bsd
+
+The `pgagroal_os_bsd` metric reports the kernel version for **BSD-based** systems (such as FreeBSD and OpenBSD).  
+Since BSD systems do not provide a patch version in the same way as Linux/macOS, the metric includes only:
+
+- `os`: The name of the operating system.
+- `major`: The major version of the kernel.
```

Comment:
kernel -> operating system

On BSD we are retrieving OS version, not stricly speaking the kernel version. We should emphasize this change everywhere.